style.css

```diff
@@ -325,3 +324,81 @@ body {
 .lang .lang-exp .no-1 span i {
   color: #acb7c3;
 }
+
+/* contact form */
+.contact-section {
+  background-color: #6070ff;
+  background-image: url(/images/contact-graph-2.png), url(/images/contact-graph-1.png), url(/images/contact-circle-1.png);
+  background-repeat: no-repeat, no-repeat, no-repeat;
+  background-size: 95px, 100px, 28px;
+  background-position: 77% 95%, 110% 3%, 40% 10%;
+  margin-top: 0;
+  padding: 3rem;
+  padding-top: 7rem;
+  border-top-left-radius: 100px 70px;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+}
+
+.contact-content h1 {
+  padding: 0;
+  margin: 0;
+  font-weight: 700;
+  font-size: 45px;
+  color: #fff;
+  text-align: center;
+  margin-bottom: 1rem;
+}
+
+.contact-content p {
+  font-weight: 400;
+  font-size: 20px;
+  color: #ebebff;
+  text-align: center;
+  line-height: 28px;
+  letter-spacing: 0.01rem;
+}
+
+.form-sections {
+  margin-top: 2rem;
+  list-style-type: none;
+  padding: 24px;
+}
```

Comment:
- Hey, there's an additional padding-left: 40px been applied to your form which is making your design look slightly different compared to the Figma design. kindly check to remove it to improve your design. see the image below to gain clarity.






![Screenshot from 2022-12-15 01-27-35](https://user-images.githubusercontent.com/59397159/207744533-66ada7fb-ab29-4ed4-8829-4ffa210a16b7.png)

Reply:
Fixed.